src/renderer/shared/ui/LabelHelpbox/LabelHelpBox.tsx

```diff
@@ -1,22 +1,29 @@
 import { PropsWithChildren } from 'react';
 
-import { BodyText, Icon } from '@renderer/shared/ui';
+import { FootnoteText, Icon } from '@renderer/shared/ui';
 import { cnTw } from '@renderer/shared/lib/utils';
 
 type Props = {
   className?: string;
+  disabled?: boolean;
 };
 
-export const LabelHelpBox = ({ className, children }: PropsWithChildren<Props>) => (
-  <div
+export const LabelHelpBox = ({ className, children, disabled = false, ...props }: PropsWithChildren<Props>) => (
```

Comment:
What kind of props might come here? Without explicit declaration in `Props` it's not clear
Maybe it's not needed at all?
Also I would add `children: string` as well 😉